cirq/iqm/iqm_device.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""IQM devices https://iqm.fi/devices"""  # TODO: PQC-5
+
+from typing import cast
+import cirq
+from cirq import devices, ops
+
+
+class Adonis(devices.Device):
+    """IQM's five-qubit superconducting device with pairwise connectivity.
+    Details: https://iqm.fi/devices
+    """
+    # TODO: PQC-5
+
+    QUBIT_DIAGRAM = "-Q-\n" \
```

Comment:
This is a first draft so i'm assuming this variable is going to be used more than once in the future. It should not be here if not.

Reply:
The idea is that this serves as class-public documentation about the qubit connectivity. Would seem silly to just copy-paste this to a docstring...

Reply:
The idea is that cirq users will be using cirq (perhaps through an IDE) or viewing from readthedocs (https://cirq.readthedocs.io/en/latest/).  They may not follow a link.  These users will need basic information about the device and whether it is what they are looking to use.  For detailed information, they can follow up in the link, but they will need basic information on what this class is.

Reply:
Not to a docstring. To the method that calls it.

Reply:
Well, I still think that this diagram would be useful for developers to have as a public variable. But this is your project and I respect that, so I will encapsulate this diagram inside the method that uses it and expose that information via a docstring.

Reply:
Done in https://github.com/quantumlib/Cirq/pull/2701/commits/d6eb9b535923873d7c880e82dd19802763e8b5e4